registries/adapters/local_openshift_adapter.go

```diff
@@ -74,71 +99,87 @@ func (r LocalOpenShiftAdapter) FetchSpecs(imageNames []string) ([]*bundle.Spec,
 		return nil, err
 	}
 
-	fqImages, err := openshiftClient.ConvertRegistryImagesToSpecs(imageNames)
+	listImages, err := openshiftClient.ListRegistryImages()
 	if err != nil {
 		log.Errorf("Failed to load registry images")
```

Comment:
general comment about error handling in bundle-lib, but I think we should try to make use of the pkg/errors . This would mean that this kind of error handling would become:
```
return errors.Wrap(err,"Failed to load registery images")
```
then the top level caller would decide what to do with the actual error. Log it , ignore, see the stacktrace ```log.Errorf("%+v", err) ``` the benefit would be reducing noise and handing control back to the consumer of the lib for this kind of thing

Reply:
No def not a big need to do it ASAP. more of a nice to have / UX consideration as you point out below